Dockerfile.build

```diff
@@ -1,4 +1,4 @@
-FROM fedora:31
+FROM fedora:32
```

Comment:
why does this need to change in this PR?

Reply:
golang.mk make commands requires golang 1.14.4+. `fedora:31` can only install golang 1.13, will have the following error:
```
vendor/github.com/openshift/build-machinery-go/make/targets/golang/../../lib/golang.mk:22: *** `go` is required with minimal version "1.14.4", detected version "1.13.15". You can override this check by using `make GO_REQUIRED_MIN_VERSION:=`.  Stop.
```

I tried to use `dnf -y update` still using golang 1.13, so used fedora:32 instead (which install golang 1.15).

Reply:
I see, thanks